### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by olenitut**
- [DEMO LINK](https://olenitut.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/watch.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@keyframes rotate {
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/watch.scss:1]: move animation into separate file

File: src/styles/blocks/watch.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    position: absolute;
+    height: $second-height;
+    width: $second-width;
+    border-radius: 10px;
+    background-color: #f1ac88ff;
+    left: calc(50% - #{$second-width / 2});
+    top: calc(50% - #{$second-height});
+    animation: rotate 60s steps(60, end) infinite;
+    transform-origin: bottom center;
+  }
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/watch.scss:48]: move repetitive styles into common selector

File: src/styles/utils/variables.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+$second-width : 10px;
+$second-height : 300px;
```
Comment (ivanchuk-yurii) [inline src/styles/utils/variables.scss:3]:
```suggestion
$second-width: 10px;
$second-height: 300px;
```

File: src/styles/blocks/watch.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+.watch {
+  position: relative;
+  list-style: none;
+
+  &__center {
+    position: absolute;
+    left: 50%;
+    right: 50%;
+    transform: translate(-50%, -50%);
+    min-width: 50px;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/watch.scss:20]: just `width`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/utils/variables.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+$second-width: 10px;
```
Comment (DarkMistyRoom) [inline src/styles/utils/variables.scss:2]:
```suggestion
$second-width: 10px;
```

**Review by DarkMistyRoom — APPROVED** [review]
Good, but be careful about code style

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**